float64 dx
float64 dy
float64 dalt
float64 dyaw
---
bool    command_status
